docs/src/ad/movement/sccm-mecm/privilege-escalation.md

```diff
@@ -1,201 +1,241 @@
 
-Step 2: Use `sccmwtf.py` to extract NAA secrets
+2. **Relaying a passive site server**
 
-A controlled computer account is needed to send the authenticated request (this is why a computer account has been created previously) to retrieve the policy, but the account to spoof doesn't need to be the same.
+As describer by [Garrett Foster](https://twitter.com/garrfoster) in this [article](https://posts.specterops.io/sccm-hierarchy-takeover-with-high-availability-7dcbd3696b43), when a passive site server is set up for high availability purpose, its machine account must be a member of the local Administrators group on the active site server. It must also be administrator on all the site system deployed in the site, including the MSSQL database. As a result, the same NTLM relaying attacks as described for primary site servers can be exploited in order to compromise the SCCM site database or gain privileged access to the HTTP API of the SMS Provider.
 
-Here, `$SCCM_MP_NetBiosName` takes the value of the SCCM Management Point server NETBIOS name.
 
-```bash
-sccmwtf.py fakepc fakepc.$DOMAIN $SCCM_MP_NetBiosName "$DOMAIN\controlledComputer$" "controlledPassword"
-```
 
-Step 3: Obtain obfuscated NAA secrets
+## Practice
 
-The obufscated NAA secrets will be saved in a local file.
+### Credential harvesting
 
-```bash
-cat /tmp/naapolicy.xml
-```
+As noted in the [Theory section](#theory), secrets can be harvested from SCCM secret policies, or Distribution Point resources.
 
-Values to decode are the long hexadecimal strings in the CDATA sections (`<![CDATA[String_here]`).
+#### Secret SCCM policies
 
-Step 4: Decode obfuscated strings
+Only **approved SCCM devices** can fetch secret policies. It is first possible to **register a device ourselves** by providing domain machine account credentials (or exploiting automatic device approval).
 
-To decode username and password use `.\DeobfuscateSecretString.exe` contained in [SharpSCCM](https://github.com/Mayyhem/SharpSCCM) or [sccmwtf](https://github.com/xpn/sccmwtf/blob/main/policysecretunobfuscate.c)
+::: tabs
 
-````powershell
-policysecretdecrypt.exe $HEX_STRING
-````
+=== UNIX-based
 
-Alternatively, [sccmhunter](https://github.com/garrettfoster13/sccmhunter) (Python) automates all the attack with, or without, an already controlled computer accounts. For this purpose, the `http` module uses the result from the `find` command and enumerates the remote hosts for SCCM/MECM enrollment web services. If it finds one, it performs [Adam Chester](https://twitter.com/_xpn_)'s attack for the specified computer account. If no account is already under control, the `-auto` flag can be indicated to create a new computer account.
+To this end, [SCCMSecrets.py](https://github.com/synacktiv/SCCMSecrets/) can be used. All secret policies associated with the default collections for the newly registered device (including NAAs, Task sequences and collection variables) will be dumped.
```

Comment:
When we indicate a tool with its URL like here, we always indicate into brackets its code langage (C#), (Python),... You can look at how it is indicate in the other code examples on the page.
Think about adding (Python) to all the SCCMSecrets references :)

Reply:
Indeed, I've added this format to the various tool references on the page